src/utils/getUnitsAmount.ts

```diff
@@ -0,0 +1,10 @@
+export function getUnintsAmount(unintsAmount: number) {
+  if (unintsAmount > 20) {
```

Comment:
Why "unitsAmount > 20"

U like say "less than 20 or equals"? @femacedogomes

Reply:
The logic is correct